cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -734,6 +734,18 @@
     </description>
   </property>
 
+  <property>
+    <name>data.storage.sql.jdbc.driver.external</name>
+    <value>true</value>
+    <description>
+      Indicates whether the JDBC driver has to be loaded from an external directory.
+      If true, then the JDBC driver directory has to be specified using
+      "data.storage.sql.jdbc.driver.directory".
+      If false, then the JDBC driver is present in the CDAP classpath.
+      This config can only be used when the storage implementation is sql.
```

Comment:
It is clearer here to say `postgresql` since we don't support general sql in 6.0

Reply:
I just copied it from the definition below for - `data.storage.sql.jdbc.driver.directory`. I will update both.